packages/client/src/views/SysAdmin/Performance/metricsQuery.ts

```diff
@@ -67,11 +67,12 @@ export const PERFORMANCE_STATS = gql`
     $populationYear: Int!
     $status: [String]!
     $event: String
+    $officeSelected: Boolean!
   ) {
     getLocationStatistics(
       locationId: $locationId
       populationYear: $populationYear
-    ) {
+    ) @skip(if: $officeSelected) {
```

Comment:
As the location stats is not rendered in case of office is selected, no point querying stats data when office is selected